gdal/ogr/ogrsf_frmts/shape/shpopen.c

```diff
@@ -834,19 +834,50 @@ SHPOpenLL( const char * pszLayer, const char * pszAccess, SAHooks *psHooks )
     return( psSHP );
 }
 
+/************************************************************************/
+/*                              SHPOpenLLEx()                           */
+/*                                                                      */
+/*      Open the .shp and .shx files based on the basename of the       */
+/*      files or either file name. It generally invokes SHPRestoreSHX() */
+/*      in case when SHP_RESTORE_SHX true config option was passed.     */
```

Comment:
The comment about SHP_RESTORE_SHX isn't appropriate in that file. Should mention bRestoreSHX instead